src/app/components/ExerciseCard/ExerciseCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import 'tailwindcss/tailwind.css';
+
+type ExerciseCardProps = {
+  thumbnail: string;
+  name: string;
+  category: string;
+};
+
+function ExerciseCard({
+  thumbnail,
+  name,
+  category,
+}: ExerciseCardProps): JSX.Element {
+  return (
+    <article className="w-32 h-44 rounded-xl grid">
+      <div className="">
+        <img
+          className="w-44 h-24 rounded-t-xl object-cover"
+          src={thumbnail}
+          alt=""
+        />
+      </div>
```

Comment:
You don't need this `div` element. If you want to force a line-break, consider adding `display: block` to your `img`.

```suggestion
        <img
          className="w-44 h-24 rounded-t-xl object-cover"
          src={thumbnail}
          alt=""
        />
```